Core/include/Acts/Propagator/detail/VolumeMaterialInteraction.hpp

```diff
@@ -73,6 +73,30 @@ struct VolumeMaterialInteraction {
         performCovarianceTransport(state.stepping.covTransport),
         navDir(state.options.direction) {}
 
+  /// @brief Constructor
+  ///
+  /// @tparam propagator_state_t Type of the propagator state
+  /// @tparam stepper_t Type of the stepper
+  ///
+  /// @param [in] vVolume The current volume
+  /// @param [in] state State of the propagation
+  /// @param [in] stepper Stepper in use
+  template <typename propagator_state_t, typename stepper_t>
+  VolumeMaterialInteraction(const Acts::Experimental::DetectorVolume* vVolume,
```

Comment:
Should this require moving all of this to `Experimental`?

Reply:
@asalzburger, I think it's a question to you

Reply:
Btw, `InteractionVolume`, which has a `Detector`/`TrackingGeometry` switch and which is already a part of the main (`MaterialInteractor` result), is not a part of the `Experimental`. 

Was it the case of "If it has a default `TrackingGeometry` behaviour then it shouldn't be in the `Experimental`"? 

I think that we should be consistent here, but placing every switch like that inside the `Experimental` may result in big chunks of the library sitting in this namespace.

Reply:
I think it's already clear when the method consumes an experimental class that it would be experimental - but I have no strong opinion here.